src/content/blog/astro-4140.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: "Astro 4.14"
+description: Astro 4.14 is available now! This release includes a first experimental version of Content Layer, experimental support for intellisense inside content files, and more.
+publishDate: "August 15, 2024"
+authors:
+  - erika
+  - matt
+coverImage: "/src/content/blog/_images/astro-4140/header_astro_4_14.webp"
+socialImage: "/src/content/blog/_images/astro-4140/og_image_4_14.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro";
+import BlogSummary from "/src/components/BlogSummary.astro";
+import frontmatterIntellisense from "/src/content/blog/_images/astro-4140/frontmatter-intellisense.png";
+
+<BlogSummary>
+  Astro 4.14 is out now! This release includes the first experimental release of Content Layer, our new super-flexible solution for managing content in Astro projects, support for intellisense inside content files, and more.
+</BlogSummary>
+
+This release includes the following highlights:
+
+- [**Experimental: Content Layer**](#experimental-content-layer)
+- [**Experimental: Intellisense inside content**](#experimental-intellisense-inside-content)
+- [**New `injectTypes` integration API**](#new-injecttypes-integration-api)
+- [**Support for metastrings in the Code component**](#support-for-metastrings-in-the-code-component)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental: Content Layer
+
+In Astro 2.0 we introduced [Content Collections](https://docs.astro.build/en/guides/content-collections/), a powerful way to manage local content in Astro projects. While this has helped hundreds of thousands of developers build content-driven sites with Astro, we've heard your feedback that you want more flexibility and power in managing your content. We're excited to announce the experimental release of the next generation of content collections in Astro: the Content Layer API.
+
+The new Content Layer API builds upon content collections, taking them beyond local files in `src/content/` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project. These new collections work alongside your existing content collections, and you can migrate them to the new API at your own pace. We think you'll want to soon, because they have lots of advantages, but there's no rush – the existing collections will continue to work as they always have.
```

Comment:
```suggestion
The new Content Layer API builds upon content collections, taking them beyond local files in `src/content/` and allowing you to fetch content from anywhere, including remote APIs. These new collections work alongside your existing content collections, and you can migrate them to the new API at your own pace. We think you'll want to soon, because they have lots of advantages, but there's no rush – the existing collections will continue to work as they always have.
```
We made a similar edit to the documentation to tighten this up. I *could* see explicitly mentioning the file system as an alternative, as that might be considered a cool thing, too. Or, I'd let the "remote APIs" shine because people will be excited for this!